packages/components/src/internal/components/entities/constants.ts

```diff
@@ -62,8 +64,38 @@ export const AssayResultDataType: EntityDataType = {
     uniqueFieldKey: 'RowId',
     dependencyText: undefined,
     filterCardHeaderClass: 'filter-card__header-purple',
+    sampleFinderCardType: 'assaydata',
```

Comment:
Is it worth having an enum for these card types and then maybe some utility methods for the various places where we do string comparisons of these types to provide better encapsulation?

Reply:
it's currently typed to be a fixed set of strings, so probably good enough?